### DISCUSSION TITLE
Add ability to rate limit by subdomain for the rate limiting plugin

### DISCUSSION THREAD

**Discussion by theundeadmonk** [Kong/kong#9954] [Ideas and feature requests]
Hi,
The basic kong rate limiting plugin does not currently support the ability to rate limit by `subdomain`
eg `foo.bar.com` rate limit by `foo`

This is useful as a lot of organizations use subdomains to achieve multi tenancy. I am happy to submit a PR for this feature if someone at Kong would grant me write access to open pull requests.

Thanks

**Comment by chronolaw** [comment]
Thanks for your concern.

I think you can fork kong repo, create a feature branch then open the PR.
